proto/comdex/auctionsV2/v1beta1/auction.proto

```diff
@@ -0,0 +1,112 @@
+syntax = "proto3";
+package comdex.auctionsV2.v1beta1;
```

Comment:
the better and accepted directory structure would be comdex.auctions.v2. Let changes this across @pratikasr @rajrohit10

Reply:
Updated